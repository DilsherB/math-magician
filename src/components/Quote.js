import React, { useEffect } from "react";

const Quote = () => {
  useEffect(() => {
    const fetchData = async () => {
      
    };
    fetchData();
  });
  return <div><h1>Hi</h1></div>;
};

export default Quote;
