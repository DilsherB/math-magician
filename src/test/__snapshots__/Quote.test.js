import renderer from "react-test-renderer";
import React, { useEffect, useState } from "react";
import Quote from "../../components/Quote";

const renderTree = (tree) => renderer.create(tree);
describe("<Quote>", () => {
  it("should render component", () => {
    expect(renderTree(<Quote />).toJSON()).toMatchSnapshot();
  });
});
