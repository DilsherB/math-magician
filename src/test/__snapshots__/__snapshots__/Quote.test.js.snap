// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Quote> should render component 1`] = `
<div
  className="d-flex align-items-center"
  style={
    Object {
      "height": "85vh",
    }
  }
>
  <div
    className="quotes w-50"
    style={
      Object {
        "margin": "auto",
      }
    }
  >
    <div
      className="center"
    >
      Loading...
    </div>
  </div>
</div>
`;
